SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialClassInfoManager.cpp

```diff
@@ -492,6 +492,11 @@ const TArray<Schema_FieldId>& USpatialClassInfoManager::GetFieldIdsByComponentId
 	return SchemaDatabase->FieldIdsArray[SchemaDatabase->ComponentIdToFieldIdsIndex[ComponentId]].FieldIds;
 }
 
+const TArray<Schema_FieldId>& USpatialClassInfoManager::GetListIdsByComponentId(Worker_ComponentId ComponentId)
+{
+	return SchemaDatabase->ListIdsArray[SchemaDatabase->ComponentIdToFieldIdsIndex[ComponentId]].FieldIds;
```

Comment:
Could a malicious update crash a server here? Not sure if it would be caught somewhere else but indexing without checks makes me nervous :D 

Perhaps `Find` in combination with check if performance is a concern

Reply:
Continued conversion about malicious updates in safe space!

Reply:
Is this concern different or the same as for the `GetFieldIdsByComponentId` function above?

Reply:
The same but I think this is ok now after the discussions in safe space, component ids are protected by the runtime providing valid data streams

Reply:
I am sticking with the convention of the function above, I don't think it reads too badly like this.